### PR TITLE
Use Openshift local cluster profile

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: ci-operator/config/crc-org/crc/crc-org-crc-main.yaml [hunk 1]
```diff
@@ -43,15 +43,15 @@ resources:
...
     workflow: code-ready-crc-e2e
 - as: integration-crc
   steps:
-    cluster_profile: gcp-3
+    cluster_profile: osl-gcp
     workflow: code-ready-crc-integration
 - as: e2e-microshift-crc
   steps:
-    cluster_profile: gcp-3
+    cluster_profile: osl-gcp
```
Comment (gbraad) [inline ci-operator/config/crc-org/crc/crc-org-crc-main.yaml:54]: `osl` stands for OpenShift Local?
Please, nly use this for technical reasons, as this is not an official abbreviation.

**Review by gbraad — APPROVED** [review]
/lgtm

Note: make sure `osl` is NOT used as an official acronym.

File: ci-operator/config/crc-org/crc/crc-org-crc-main.yaml (hunk 1 above)
Comment (praveenkumar) [reply ci-operator/config/crc-org/crc/crc-org-crc-main.yaml:54]: yes, it is not offical abbreviation but need to use a short form for technical reason.

Comment (cfergeau) [reply ci-operator/config/crc-org/crc/crc-org-crc-main.yaml:54]: Why not `crc`?

**Pull request merged**